eclair-core/src/main/scala/fr/acinq/eclair/blockchain/fee/FeeProvider.scala

```diff
@@ -105,9 +110,14 @@ object FeeratePerKw {
 }
 
 /**
- * Fee rates in satoshi-per-kilo-bytes (1 kb = 1000 bytes).
+ * Fee rates in satoshis-per-kilo-bytes (1 kb = 1000 bytes).
  * The mempoolMinFee is the minimal fee required for a tx to enter the mempool (and then be relayed to other nodes and eventually get confirmed).
  * If our fee provider doesn't expose this data, using its biggest block target should be a good enough estimation.
+ *
+ * The mempool contains pending transactions. Fewer pending transactions means lower fees.
+ * The block targets of 1, 2, 6, 12, 36, 72, 144, 1008 roughly correspond to
+ * time intervals of 10 minutes, 20 minutes, 1 hour, 2 hours, 6 hours, 12 hours, 1 day, 1 week respectively.
+ * These are the time intervals eclair expects the transaction to take to be included into a block.
```

Comment:
Same, please remove this comment, it's really unnecessary. It's basic bitcoin trivia that the target block time is 10 minutes, so people can compute those themselves if they're interested.

Reply:
OK. Removed. The correspondence between blocks and time wasn't obvious to me, but I am a newbie.

Reply:
The comment hasn't been removed in the last version of the PR?

Reply:
Sorry - removed it this time.